eng/Generate.ps1

```diff
@@ -84,19 +84,20 @@ if (!($Exclude -contains "TestServer"))
     }
 }
 
-$llcArgs = "--low-level-client=true --security=AzureKey --security-header-name=Fake-Subscription-Key"
+$llcArgs = "--low-level-client=true --security=AzureKey --security-header-name=Fake-Subscription-Key --modelerfour.group-parameters=false"
```

Comment:
Looks like you don't need this change anymore with your previous PR

Reply:
Correct!  I will rebase on top of that change and remove this.

Reply:
Merged the parent branch in and undid this change.